auth/service.cc

```diff
@@ -30,6 +30,7 @@
 #include "db/functions/function_name.hh"
 #include "exceptions/exceptions.hh"
 #include "log.hh"
+#include "maintenance_mode.hh"
```

Comment:
>  auth/service: don't create system_auth keyspace when creating maintenance port

More context should be provided by the commit message: the fact that there will be two auth services, and only the new auth service will not create system_auth.